app/assets/stylesheets/home.css.scss

```diff
@@ -1,34 +1,34 @@
-// .overlay-container .overlay {
-//   position: absolute;
-//   top: 0;
-//   bottom: 0;
-//   left: 0;
-//   right: 0;
-//   padding: 0.4rem;
-//   background-color: rgba(255, 255, 255, 0.01);
-//   text-align: center;
-//   font: 0/0 a; /* Remove the gap between inline(-block) elements */
-// }
-//
-// .overlay-container .overlay:before {
-//   content: ' ';
-//   display: inline-block;
-//   vertical-align: middle;
-//   height: 100%;
-// }
-//
-// .overlay-container .overlay span {
-//   font: 22px/1 Georgia, sans-serif; /* Reset the font property */
-//   color: #1E90FF;
-//   display: inline-block;
-//   font-weight: bold;
-//   vertical-align: middle;
-// }
-//
-// .h1 {
-//   position: absolute;
-//   left: 200px;
-// }
+.overlay-container .overlay {
+  position: absolute;
```

Comment:
Properties should be ordered background-color, bottom, font, left, padding, position, right, text-align, top